cli/cmd/turbo/main.go

```diff
@@ -19,6 +22,19 @@ func nativeRunWithArgs(argc C.int, argv **C.char) C.uint {
 	for i, arg := range unsafe.Slice(argv, arglen) {
 		args[i] = C.GoString(arg)
 	}
+
 	exitCode := cmd.RunWithArgs(args, turboVersion)
 	return C.uint(exitCode)
 }
+
+//export nativeRunWithTurboState
+func nativeRunWithTurboState(turboStateString string) C.uint {
```

Comment:
just making sure i understand. Is this new function is called by the Rust shim?

Reply:
Yep it is.